bin/run.js

```diff
@@ -47,6 +47,7 @@ if(argv.version) {
             exclude: argv['display-exclude'],
             colors: argv['colors'],
             stats: argv['stats'],
+            timeout: argv['timeout'],
```

Comment:
Could you add it to the schema as well?
https://github.com/trivago/parallel-webpack/blob/master/schema.json